### PR TITLE
windows: credentials implementation

### PR DESCRIPTION
**Pull request by zaucy**
Release Notes:

- N/A

### CODE REVIEW

File: crates/gpui/src/platform/windows/platform.rs [hunk 1]
```diff
@@ -19,24 +21,19 @@ use itertools::Itertools;
...
     },
 };
 
-use crate::{
-    Action, AnyWindowHandle, BackgroundExecutor, ClipboardItem, CursorStyle, ForegroundExecutor,
-    Keymap, Menu, PathPromptOptions, Platform, PlatformDisplay, PlatformInput, PlatformTextSystem,
-    PlatformWindow, Task, WindowAppearance, WindowParams, WindowsDispatcher, WindowsDisplay,
-    WindowsTextSystem, WindowsWindow,
-};
+use crate::*;
```
Comment (zaucy) [inline crates/gpui/src/platform/windows/platform.rs:36]: did this to hopefully not conflict with the other PR with this change

File: crates/gpui/src/platform/windows/util.rs [hunk 2]
```diff
@@ -68,3 +68,7 @@ pub(crate) unsafe fn set_window_long(
         SetWindowLongW(hwnd, nindex, dwnewlong as i32) as isize
     }
 }
+
+pub(crate) fn windows_credentials_target_name(url: &str) -> String {
+    format!("zed:url={}", url)
```
Comment (zaucy) [inline crates/gpui/src/platform/windows/util.rs:73]: chose this format to follow other applications in my credentials manager (such as the gh cli)

**Review by mikayla-maki — APPROVED** [review]

**Comment by mikayla-maki** [comment]
It seems there are still some conflicts :)

**Comment by zaucy** [comment]
fixed!

**Pull request merged**